.github/workflows/ci.yml

```diff
@@ -315,7 +324,7 @@ jobs:
           DOCS_GIT_PSW: ${{ secrets.UPBOUND_BOT_GITHUB_PSW }}
 
       - name: Promote Artifacts in S3 and Docker Hub
-        if: github.ref == 'refs/heads/master' && env.AWS_USR != '' && env.DOCKER_USR != ''
+        if: github.ref == 'refs/heads/master' && env.AWS_USR != '' && env.DOCKER_USR != '' && env.UPBOUND_ROBOT_USR != ''
```

Comment:
Does this mean only images promoted in `master` will be pushed to Upbound Registry? Don't we need changes in `promote.yml` as well?

Reply:
@muvaf this means that we will have the exact same tags in dockerhub and the Upbound Registry (i.e. on `master` we publish the version tag and the `master` tag that both point to the same manifest). You are totally right that we need to add the login step to `promote` CI as well -- thanks for catching, updating now :)